src/main/java/sirius/biz/web/BizController.java

```diff
@@ -150,11 +152,30 @@ protected void load(WebContext ctx, Entity entity) {
      */
     protected void load(WebContext ctx, Entity entity, Column... properties) {
         Set<String> columnsSet = Arrays.stream(properties).map(Column::getName).collect(Collectors.toSet());
+        load(ctx, entity, columnsSet);
+    }
+
+    private void load(WebContext ctx, Entity entity, Set<String> properties) {
```

Comment:
Hauptsächlich weil die eine Methode bereits mit einem set gearbeitet hat und es recht einfach war die andere auch in ein set umzuwandeln